src/dotnet/CommandResolution/ProjectToolsCommandResolverPolicy.cs

```diff
@@ -0,0 +1,18 @@
+using Microsoft.DotNet.Cli.Utils;
+
+namespace Microsoft.DotNet.Cli.CommandResolution
+{
+    public class ProjectToolsCommandResolverPolicy : ICommandResolverPolicy
+    {
+        public CompositeCommandResolver CreateCommandResolver()
+        {
+            var defaultCommandResolverPolicy = new DefaultCommandResolverPolicy();
+            var compositeCommandResolver = defaultCommandResolverPolicy.CreateCommandResolver();
+            var packagedCommandSpecFactory = new PackagedCommandSpecFactory();
+
+            compositeCommandResolver.AddCommandResolver(new ProjectToolsCommandResolver(packagedCommandSpecFactory));
```

Comment:
Because the ProjectToolsCommandResolver needs Microsoft.Build package, which is only for net46 and netstandard15. The default policy lives in Cli.Utils, which has net451 and netstandard13. So, net46 is not compatible with those TFMs. So, I had to move that class to somewhere where I could depend on Microsoft.Build.

Makes sense?